deploy/example_catalog/cr-app-tensorflowcpu.json

```diff
@@ -0,0 +1,67 @@
+{
+    "apiVersion": "kubedirector.hpe.com/v1beta1",
+    "kind": "KubeDirectorApp",
+    "metadata": {
+        "name" : "jupyter-notebook"
+    },
+
+    "spec" : {
+        "systemdRequired": true,
+        "config": {
+            "roleServices": [
+                {
+                    "serviceIDs": [
+                        "ssh",
+                        "jupyter-nb"
+                    ],
+                    "roleID": "controller"
+                }
+            ],
+            "selectedRoles": [
+                "controller"
+            ]
+        },
+        "label": {
+            "name": "Tensorflow 1.14 with jupyter notebook & centos7x",
+            "description": "TensorFlow1.14 with centos7x",
+            "AIML_category" : "Notebook"
+        },
+        "distroID": "bluedata/jupyter-notebook",
+        "version": "1.8",
```

Comment:
version to be 1.0. I don't think we need configSchemaVersion, verify existing apps

Reply:
we should remove **bluedata** from distroID